src/laser_proc.cpp

```diff
@@ -27,20 +27,146 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/* 
+/*
  * Author: Chad Rockey
  */
 
-#include <laser_proc/LaserProcROS.h>
+#include <limits>
+#include <vector>
+
+#include "laser_proc/laser_proc.hpp"
+
+using namespace laser_proc;  // NOLINT
```

Comment:
Instead of doing this with a `NOLINT`, can we instead just do:

```
namespace laser_proc
{
```

At the top here, and close out the namespace at the bottom of the file?

Reply:
This comment is still open.